doc/advanced-configuration/release-options.md

```diff
@@ -0,0 +1,226 @@
+When building a MongooseIM release from source code, the initial configuration files are generated with options taken from the `vars-toml.config` file found in the `[MongooseIM root]/rel/` directory.
+You can change the values in this file to affect the resulting `vm.args` and `mongooseim.toml` files.
+
+The file contains erlang tuples terminated with period ('.'). For users not familiar with Erlang syntax, here is a quick cheat sheet:
+
+* Each config option (key and value) is a tuple. Tuples are (Erlangers, forgive us the simplification) other Erlang terms separated with commas and enclosed in curly brackets ({}).
+* Tuples (at least the top-level ones) in `vars.config` are always 2-element.
+* The first element of each tuple is the name (Erlang atom).
+* The second element is a quoted string. Any quotes (`"`) inside the string should be escaped with a backslash (`\`).
+
+There are two types of options: parameters and blocks:
+
+* a **parameter** is inserted into the value of an already defined option.
+Parameters are mandatory - a valid value has to be provided.
+* a **block** can be an empty string, one line or multiple lines, defining zero, one or more options.
+Blocks are optional - the default is an empty string.
+
+# vm.args options
+
+These options are inserted into the `rel/files/vm.args` template.
+
+## node_name
+
+* **Type:** parameter
+* **Option:** value of `-sname` in [vm.args](../../advanced-configuration#options)
+* **Syntax:** Erlang node name: `name@host`
+* **Example:** `{node_name, "mongooseim@localhost"}.`
+
+## highload_vm_args
+
+* **Type:** block
+* **Option:** arguments in [vm.args](../../advanced-configuration#options): `+K`, `+A`, `+P`, `-env ERL_MAX_PORTS`
+* **Syntax:** command-line arguments
+* **Example:** `{highload_vm_args, "+P 10000000 -env ERL_MAX_PORTS 250000"}.`
+
+# TOML Options
+
+These options are inserted into the `rel/files/mongooseim.toml` template.
+
+## hosts
+
+* **Type:** parameter
+* **Option:** [`general.hosts`](../../advanced-configuration/general#generalhosts)
+* **Syntax:** comma-separated list of strings
+* **Example:** `{hosts, "\"localhost\", \"domain2\""}.`
+
+## host_config
+
+* **Type:** block
+* **Option:** [`host_config`](../../advanced-configuration/host_config.md)
```

Comment:
```suggestion
* **Option:** [`host_config`](../../advanced-configuration/host_config)
```
I've run it locally and was not working. I think we add `.md` for absolute paths.